blogs/pyladies-soflo.json

```diff
@@ -0,0 +1,20 @@
+{
+  "title": "PyLadies SoFlo YouTube channel",
+  "type": "YouTube channel",
```

Comment:
`type` should be `"type": "youtube"` so that the algorithm captures it correctly :)